CMakeLists.txt

```diff
@@ -217,10 +221,6 @@ endif()
 
 feature_summary(WHAT ALL)
 
-add_library(opus ${opus_sources} ${opus_sources_float})
```

Comment:
revert this change for now

Reply:
The public headers have to be included here or they are not installed into the built framework.

Reply:
I see, then leave it and I add the other headers at a later stage.

Reply:
Thanks!